Rakefile

```diff
@@ -0,0 +1,39 @@
+#
+# constants
+#
+CHEF_VERSION = '11.16.4'
```

Comment:
This is good for us but might not be apropos for a community cookbook-- assuming the cookbook works under Chef 12! But I think it would be a good candidate for addressing this later (after we get our own stuff working).